src/main/java/com/google/firebase/FirebaseApp.java

```diff
@@ -171,39 +171,36 @@ public static FirebaseApp getInstance(@NonNull String name) {
   }
 
   /**
-   * Initializes the default {@link FirebaseApp} instance. Same as {@link
-   * #initializeApp(String)}, but uses the name {@link #DEFAULT_APP_NAME}.
-   * Also, uses the {@link FirebaseOptions} values set defined by the JSON in
-   * {@code FIREBASE_CONFIG_ENV_VAR}. If the string value in {@code FIREBASE_CONFIG_ENV_VAR}
-   * starts with `{`, it is parsed as a JSON object otherwise it is treated
-   * as a file name, and the json is parsed from that file. 
-   * If the {code FIREBASE_CONFIG_ENV_VAR} is not set, an empty option set is used.
-   * Whether or not the options are populated in {@code FIREBASE_CONFIG_ENV_VAR}, 
-   * the Application Default Credentials are used when initializing the app like this.
+   * Initializes the default {@link FirebaseApp} instance using default credentials and
```

Comment:
Incomplete sentence here. Make sure the content here is the source of truth for the API docs published on Firebase devsite.